frame/authority-discovery/src/lib.rs

```diff
@@ -18,12 +18,12 @@
 //! # Authority discovery module.
 //!
 //! This module is used by the `client/authority-discovery` to retrieve the
```

Comment:
If I am not mistaken `current_authorities` and `next_authorities` would be called by the parachain logic. I would suggest either amending the comment or removing it all together.